irohad/peer_service/peer_service.cpp

```diff
@@ -14,45 +14,74 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+#include <numeric>
+#include <random>
+#include <unordered_set>
+#include <exception>
+
 #include <peer_service/peer_service.hpp>
 
-// TODO
 namespace peer_service {
 
   void initialize() {
```

Comment:
That is a very strange function to understand. Maybe if you classes it will be more clear